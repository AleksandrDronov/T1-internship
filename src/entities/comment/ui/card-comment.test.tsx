import renderer from "react-test-renderer";
import { CardComment } from ".";

describe("CardComment", () => {
  const comment = {
    id: 1,
    user: { username: "omottley2h" },
    body: "I cannot believe how I found you, this is so pretty.",
  };
  test("renders correctly", () => {
    const tree = renderer.create(<CardComment comment={comment} />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test("renders correctly with className", () => {
    const tree = renderer
      .create(<CardComment className="test" comment={comment} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
