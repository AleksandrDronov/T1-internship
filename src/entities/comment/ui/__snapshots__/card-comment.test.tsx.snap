// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardComment renders correctly 1`] = `
<div
  aria-label="Comment card"
  className="container"
>
  <div
    className="header"
  >
    <p
      className="text b10-web black left"
    >
      @
      omottley2h
    </p>
    <img
      alt=""
      className="icon"
      src=""
    />
  </div>
  <p
    className="text b11-web gray left"
  >
    I cannot believe how I found you, this is so pretty.
  </p>
</div>
`;

exports[`CardComment renders correctly with className 1`] = `
<div
  aria-label="Comment card"
  className="test container"
>
  <div
    className="header"
  >
    <p
      className="text b10-web black left"
    >
      @
      omottley2h
    </p>
    <img
      alt=""
      className="icon"
      src=""
    />
  </div>
  <p
    className="text b11-web gray left"
  >
    I cannot believe how I found you, this is so pretty.
  </p>
</div>
`;
